test/basic-querying.test.js

```diff
@@ -7,13 +7,16 @@
 'use strict';
 
 /* global getSchema:false, connectorCapabilities:false */
-var should = require('./init.js');
 var async = require('async');
-var db, User;
+var bdd = require('./helpers/bdd-if');
+var should = require('./init.js');
+var uuidV4 = require('uuid/v4');
+
+var db = getSchema(), User;
+var isCassandraConnector = db.connector.name === 'cassandra';
```

Comment:
I agree.  I'd defer until LB experts to come up with a right design for that, hopefully soon.